src/Tools/ILAsm/IlAsmDeploy.csproj

```diff
@@ -43,7 +43,7 @@
   </PropertyGroup>
 
   <Target Name="_PublishILAsm"
-          BeforeTargets="AfterBuild"
+          AfterTargets="Build"
```

Comment:
Why did this change?

Reply:
Working around a bad merge in another PR that broke the build. Going to revert before I merge.